Tests/SuperLinq.Async.Test/HasDuplicatesTest.cs

```diff
@@ -0,0 +1,371 @@
+﻿namespace Test.Async;
+
+public class HasDuplicatesTest
+{
+	[Fact]
+	public async Task When_Asking_For_Duplicates_On_Sequence_Without_Duplicates_Then_False_Is_Returned()
+	{
+		await using var asyncSequence = AsyncSuperEnumerable.From(
```

Comment:
Follow the `TestSequence` tests of the sync version; `await Task.Delay()` is not necessary for tests.
```cs
await using var sequence = TestSequence.Of(...);
```